<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Key</class>
 <widget class="QDialog" name="Key">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Уникальный ключ приложения</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../media/license.qrc">
    <normaloff>:/icon/add_key.ico</normaloff>:/icon/add_key.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ключ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_key_app"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save_key">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Запрос на получение  ключа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send_request_key">
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Время отправки запроса</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="minimumDate">
        <date>
         <year>2000</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="maximumTime">
        <time>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
       <property name="minimumTime">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::HourSection</enum>
       </property>
       <property name="displayFormat">
        <string>h:mm </string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
       <property name="time">
        <time>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_error_text">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../media/license.qrc"/>
 </resources>
 <connections/>
</ui>
